content/doc/upgrade-guide/2.107.adoc

```diff
@@ -12,18 +12,25 @@ Each section covers the upgrade from the previous LTS release, the section on 2.
 
 ==== Whitelist for Remoting and XStream (JEP-200)
 
+https://jenkins.io/blog/2018/01/13/jep-200/[Announcement from Jan 13],
 link:https://issues.jenkins-ci.org/browse/JENKINS-47736[JENKINS-47736],
-https://github.com/jenkinsci/jep/blob/master/jep/200/README.adoc[JEP-200 proposal and design],
-https://jenkins.io/blog/2018/01/13/jep-200/[announcement blog post from January 13]
+https://github.com/jenkinsci/jep/blob/master/jep/200/README.adoc[JEP-200 proposal and design]
 
 To prevent further security issues related to unsafe deserialization of Java objects, we switched to a whitelist for Remoting (the communication library mostly used between agents and master) and XStream (the XML serialization library).
 This change means that only classes considered to be safe to (de)serialize will now be (de)serialized.
 
 See https://jenkins.io/blog/2018/01/13/jep-200/#for-jenkins-administrators[the announcement blog post] for advice on how to upgrade Jenkins to a version that includes this change.
 
-Many plugins require updates to be compatible with this new restriction, otherwise they will fail to load or save data on disk, or fail to execute commands e.g. during a build.
-For a list of known affected plugins, see https://wiki.jenkins.io/display/JENKINS/Plugins+affected+by+fix+for+JEP-200[this wiki page].
-For a list of issues related to JEP-200, see https://issues.jenkins-ci.org/issues/?jql=labels%20%3D%20JEP-200[the label JEP-200 in the issue tracker].
+Potential impact:
+
+* Many plugins require updates to be compatible with this new restriction
+* For a list of known affected plugins, see https://wiki.jenkins.io/display/JENKINS/Plugins+affected+by+fix+for+JEP-200[this wiki page].
```

Comment:
"For a list of [plugins], see [wiki]" is not a "potential impact". This list makes no sense as a list.